tensorzero-core/src/variant/dicl.rs

```diff
@@ -620,7 +686,49 @@ impl DiclConfig {
         })
     }
 
-    fn prepare_request<'a, 'request>(
+    /// Prepare request message for DICL (stringify template content blocks, then delegate to chat_completion)
```

Comment:
can you add a bit more documentation of why? I don't think it's obvious without some thought or going to find the call site